site/src/component/SideBar/Sidebar.scss

```diff
@@ -74,7 +74,7 @@
   width: 5vw;
 
   .sidebar-links {
-    margin-top: 4vh;
+    margin-top: 2vh; 
```

Comment:
Does this affect the standard desktop view as well?  I don't see this inside the media query below for mobile devices?